collectors/health.go

```diff
@@ -615,6 +645,35 @@ func (c *ClusterHealthCollector) collect() error {
 	}
 	c.RemappedPGs.Set(remappedPGs)
 
+	scrubbingPGs     := float64(0)
+	deepScrubbingPGs := float64(0)
+	for _, pg := range stats.PGMap.PGsByState {
+		matched, err := regexp.MatchString(`scrubbing$`, pg.StateName)
```

Comment:
One small change I'd request here is to check if "scrubbing" exists in the state name but "deep" doesn't and use that to categorize them into shallow and deep scrubs instead of checking for the exact position of "scrubbing" substring in the string. Newer Ceph versions can change the placement of these states inside the state name, and our code would break if that happens.